base/util_testing.go

```diff
@@ -430,6 +430,7 @@ func setTestLogging(logLevel LogLevel, caller string, logKeys ...LogKey) (teardo
 		return func() {
 			if caller != "" {
 				Infof(KeyAll, "%v: Reset logging", caller)
+				FlushLoggerBuffers()
```

Comment:
AFAIK this `FlushLoggerBuffers` function is a special-case only for the buffered/memory loggers (which aren't initialized by tests, with the exception of `TestLogFlush`) - seems overkill to run it for all tests just to cover one case.